src/pages/Subscription/Subscription.jsx

```diff
@@ -0,0 +1,16 @@
+
+export default function SubscriptionPage(){
+	return (
+		<Fragment>

```

Comment:
I notice that you had used fragments [here](https://github.com/MarcelloAragoni/ping/pull/3/files#diff-19838ebe808d26ebc0b6c9e51de8c4caa791f6bc421017b1d2c75ba37a7c985cR3), which is perfect that way, so let's use the same format here to be consistent in our code.